packages/next/build/utils.ts

```diff
@@ -1120,6 +1120,17 @@ export function getRawPageExtensions(pageExtensions: string[]): string[] {
   )
 }
 
+export function isPage(
+  filename: string,
+  pagesDir: string,
+  pageExtensions: string[]
```

Comment:
```suggestion
  pageExtensions: NextConfigComplete["pageExtensions"]
```

So it's resilient for future changes.